subsys/usb/class/mass_storage.c

```diff
@@ -44,9 +44,9 @@
 #include <usb/usb_common.h>
 #include <usb_descriptor.h>
 
-#define SYS_LOG_LEVEL CONFIG_SYS_LOG_USB_MASS_STORAGE_LEVEL
-#define SYS_LOG_DOMAIN "usb/msc"
-#include <logging/sys_log.h>
+#define LOG_LEVEL CONFIG_USB_MASS_STORAGE_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(usb_mass_storage)
```

Comment:
I wonder would it be a bit long prefix for other logs to fit

Reply:
changed it to "usb_msc"